### ISSUE TITLE
'aria-hidden' cannot be used in <React.fragment> in the dropdown menu.

### ISSUE THREAD

**Issue by Kimhansoool** [radix-ui/primitives#3211]
### Current Behavior
When migration to Next15 and React19, an error appears saying that 'aria-hidden' cannot be used in <React.fragment>.
After searching the radix-ui menu components package code, I found that 'aria-hidden' was added whild return hideothers in MenuRootContentModal.

![image](https://github.com/user-attachments/assets/88739f90-4e22-417e-a638-18f49dc5e4d2)

![image](https://github.com/user-attachments/assets/5265e5ed-027e-4f46-b295-2471d0b8b9cf)
https://github.com/radix-ui/primitives/blob/74b182b401c8ca0fa5b66a5a9a47f507bb3d5adc/packages/react/menu/src/Menu.tsx#L258

![image](https://github.com/user-attachments/assets/db2645e7-4abd-4492-8936-4e422a835306)
https://github.com/theKashey/aria-hidden/blob/52c7374abb9fb0a348c63b9ff613a575868cec62/src/index.ts#L174

### Reproducible example
Using dropdown menu package in next15...

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |   dropdown menu       |   ^2.1.1      |
| React            | n/a     |   19.0.0-rc-45804af1-20241021      |
| Next            | n/a     |   15.0.1      |
| Browser          |   chrome      |         |
| Assistive tech   |         |         |
| Node             | n/a     |   20.12.2       |
| npm/yarn         |   pnpm      |    8.14.1     |
| Operating System |  window       |    11     |

**Comment by Kimhansoool** [comment]
#3229

**Comment by bien-phillip** [comment]
<img width="1069" alt="image" src="https://github.com/user-attachments/assets/d7f4191e-1dea-4982-8ff3-000a74b4f22b" />

I'm facing this same problem, too. I just updated all radix-ui packages. But it's not fix. Please give me any advices.
